### ISSUE TITLE
Party chat emits errors in the logs when used

### ISSUE THREAD

**Issue by kharidiron** [StarryPy/StarryPy3k#97]
sample error:
```
2017-06-17 23:20:00 - ERROR - starrypy.plugin_manager # Exception encountered in plugin on action: chat_sent
Traceback (most recent call last):
  File "/data/StarryPy3k/plugin_manager.py", line 45, in do
    if not (yield from p(packet, connection)):
  File "/data/StarryPy3k/plugins/chat_enhancements.py", line 132, in on_chat_sent
    name=sender)
  File "/data/StarryPy3k/utilities.py", line 343, in send_message
    return asyncio.ensure_future(connection.send_message(*messages, **kwargs))
AttributeError: 'NoneType' object has no attribute 'send_message'
```

**Comment by kharidiron** [comment]
The offending line: https://github.com/StarryPy/StarryPy3k/blob/master/plugins/chat_enhancements.py#L129

It looks like the social spies p.connection object is not existing, and that's causing the error.

**Comment by medeor413** [comment]
Fixed in #99.
